Owl/README.md

```diff
@@ -3,19 +3,18 @@
 <img src="screenshots/collage_header.png" alt="Owl collage"/>
 
 ## Introduction
-Owl is an educational app that uses Material Design components and [Material Theming](https://material.io/design/material-theming) to create an energetic, motivational brand experience.
-This project is an Android implementation of [Owl](https://material.io/design/material-studies/owl.html), a Material study meant to showcase componentry and theming using the [Material Components for Android library](https://github.com/material-components/material-components-android).
+This project is an Android implementation of [Owl](https://material.io/design/material-studies/owl.html), a Material Study showcasing the possibilities of using Material Theming and Material Components for Android.
 
 ## Screenshots
 <img src="screenshots/owl_demo.gif" height="400" alt="Owl sample demo"/>
 
 ## Material Theming
-Owl employs material theming to reflect the energy and excitement of learning a new skill, using a bold aesthetic and unfilled shapes that invite the user to populate them with new content and courses. The app customizes [color](https://material.io/develop/android/theming/color/), [shape](https://material.io/develop/android/theming/shape/) and [typography](https://material.io/develop/android/theming/typography/) to achieve this.
+Owl uses Material Theming to customize the app’s [color](https://material.io/develop/android/theming/color/), [shape](https://material.io/develop/android/theming/shape/) and [typography](https://material.io/develop/android/theming/typography/) to achieve this.
 
 ### Color
 <img src="screenshots/color_header.png" width="700" alt="Owl color theming"/>
 
-Owl has three primary colors. Each color is used to create a distinct visual theme for each section. See [colors.xml](https://github.com/material-components/material-components-android-examples/blob/develop/Owl/app/src/main/res/values/color.xml) and how they are used in [light](https://github.com/material-components/material-components-android-examples/blob/develop/Owl/app/src/main/res/values/theme.xml#L58-L86) and [dark](https://github.com/material-components/material-components-android-examples/blob/develop/Owl/app/src/main/res/values-night/theme.xml) themes.
+Owl has three primary colors. Each color is used to create a distinct visual theme for each section. See [colors.xml](https://github.com/material-components/material-components-android-examples/blob/develop/Owl/app/src/main/res/values/color.xml) and how they are used in [default](https://github.com/material-components/material-components-android-examples/blob/develop/Owl/app/src/main/res/values/theme.xml#L58-L86) and [dark](https://github.com/material-components/material-components-android-examples/blob/develop/Owl/app/src/main/res/values-night/theme.xml) themes.
```

Comment:
`[colors.xml]` should be `[color.xml]`

Not from your changes, but might as well fix it while we're updating!

Reply:
What do you think about adding a bit more to `and how they are used..`?

Like: `See color.xml for the full palette and how it is applied in default and dark themes.`?

Reply:
Updated this paragraph to read "Owl has three primary colors which are used to create distinct visual themes for each section. See color.xml for the full color scheme and how colors are applied across default and dark themes."